### DISCUSSION TITLE
Not working Bluetooth and display in ttgo-lora32-v2.1

### DISCUSSION THREAD

**Discussion by wotan999** [SQ9MDD/TTGO-T-Beam-LoRa-APRS#61] [Q&A]
Hi
I need help, bluetooth is not working in 2.1 version. I have tried to activate the option via WWW and also I have tried to add the option ¨-D ENABLE_BLUETOOTH¨ in platformio, it has activated it but it does not work well and the WWW stops working.

The display does not work in this version, but it works if I use firmware v2 removing the option ¨Display dimmer enabled¨

Thanks

David, EA7JYN

**Comment by mattbk** [comment]
Is it related to this? https://github.com/SQ9MDD/TTGO-T-Beam-LoRa-APRS/issues/47
